docs/demos/podman-ai-lab-to-rhoai/podman-ai-lab-to-rhoai.md

```diff
@@ -0,0 +1,447 @@
+
+    ![RHOAI Name Project](img/rhoai_name_project.png)
+
+4. We'll now create a workbench where we can upload a Jupyter notebook to ingest data into the Elasticsearch vector DB. We'll then test it out by querying for some data. Click the **Create a workbench** button.
+
+    ![RHOAI Create Workbench](img/rhoai_create_workbench.png)
+
+5. Name the workbench *elastic-vectordb-workbench*, select a *Standard Data Science* notebook, and select a *Medium* size.
+
+    ![RHOAI Setup Workbench 1](img/rhoai_workbench_1.png)
+
+    You'll also want to set two environment variables so we can connect to Elasticsearch from the notebook. 
+
+    - **CONNECTION_STRING** - Copy the CLUSTER-IP and PORT from this oc command
+
+        ```
+        oc get service elasticsearch-sample-es-http -n elastic-vectordb
+        ```
+      
+        Add the **CONNECTION_STRING** key/value as a ConfigMap environment variable.
+
+    - **PASSWORD** - Create a secret environment variable with the Elasticsearch secret value.
+        ```
+        oc get secret elasticsearch-sample-es-elastic-user -n elastic-vectordb -o jsonpath="{.data['elastic']}" | base64 -d > elastic_pass.txt
+        ```
+
+        Add the **PASSWORD** key/value as a Secret environment variable. The password is in the **elastic_pass.txt** file that was created by the above oc command.
+
+    ![RHOAI Create Workbench](img/rhoai_workbench_2.png)
+
+    **NOTE:** *You can delete the elastic_pass.txt file that you got the password from after you add it to the environment variable.*
+
+    Click on the **Create Workbench** button. Your workbench should start in a few minutes.
+
+6. Open your workbench after it has started and login.
+    
+    **Note:** *If you have insufficient resources to start a medium container size then stop the workbench and change the workbench to start as a small container size.*
+
+7. Upload or import the ./notebooks/Langchain-ElasticSearchVector-ingest.ipynb notebook to your workbench.
+
+    ![RHOAI Workbench Notebook](img/vector_ingest_notebook_upload.png)
+
+8. Run the first 3 cells (*Shift + Enter in each cell to run*). Make sure you can connect to your Elasticsearch cluster.
+
+    ![Notebook Execution - First Section](img/verify_elastic_connection.png)
+
+9. Continue to run through each cell while reading through what is occurring in each one. The *Create the index and ingest the documents* cell is where all of the websites and pdfs are stored with embeddings into the Elasticsearch vector database.
+
+    This cell will take a while to run. Good time for a coffee break.
+
+    ![Notebook Execution - Second Section](img/ingest_data_cell.png)
+
+10. After all of the data is stored into our vector database we can directly query it. Run the last 2 cells to make sure the data was stored successfully and we get back the results we expect.
+
+    ![Notebook Execution - Third Section](img/query_data_cell.png)
+
+## Deploy s3 Storage (Minio)
```

Comment:
There is already a Minio deployment recipe on the site, maybe you could point to it.
This should also be optional, if people don't already have it or other S3 solution deployed.

Reply:
Made the section optional, included the link to the Mino installation instructions on ai-on-openshift site.